src/sentry/static/sentry/app/integrationPopupViews/awsLambdaProjectSelect.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
Can we move this and `pipeline` out of `app` and into... somewhere else?

maybe `app/views/integrations`? idk